### PR TITLE
Fix CodeAnalysis analyzerdata includes

### PR DESCRIPTION
**Pull request by ViktorHofer**
Additionally format code.

cc @stephentoub

### CODE REVIEW

**Comment by stephentoub** [comment]
What is the actual fix here?  It's hard to see what's meaningful vs all of the formatting changes.

**Comment by ViktorHofer** [comment]
I was in the middle of fixing the commits. Please take another look. Commit 2 is the bugfix.

File: src/Microsoft.DotNet.CodeAnalysis/build/Microsoft.DotNet.CodeAnalysis.targets [hunk 1]
```diff
@@ -20,14 +20,16 @@
...
+    <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata.$(TargetFramework)" />
+    <!-- As of writing possible values: .netstandard, .netframework, .netcoreapp -->
+    <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata$(TargetFrameworkIdentifier.ToLower())" />
 
     <!-- Add the necessary valid Pinvoke mapping file -->
     <AdditionalFiles Include="$(ValidPInvokeMappings)" Condition="'$(ValidPInvokeMappings)' != ''" />
 
     <!-- Disable any analyzers that should not run -->
     <DisabledAnalyzers Condition="'$(IsTestProject)' == 'true' or '$(TargetsUnix)' == 'true' or '$(EnablePInvokeAnalyzer)' != 'true'" Include="PInvokeAnalyzer"/>
-    <DisabledAnalyzers Condition="'$(TargetsWindows)' == 'true' and ($(TargetGroup.Contains('net46')) or $(TargetGroup.Contains('net45')))" Include="PInvokeAnalyzer" />
```
Comment (stephentoub) [inline src/Microsoft.DotNet.CodeAnalysis/build/Microsoft.DotNet.CodeAnalysis.targets]: Have we searched the rest of arcade for places that might still be erroneously using TargetGroup?

Comment (ViktorHofer) [reply src/Microsoft.DotNet.CodeAnalysis/build/Microsoft.DotNet.CodeAnalysis.targets]: I assumed @Anipik did that when he removed the property. We have one more hit (https://github.com/dotnet/arcade/blob/f5f3e04924d149ce08fcf9ac232bda3280b72cff/src/Microsoft.DotNet.ApiCompat/build/Microsoft.DotNet.ApiCompat.targets#L28) which is unproblematic as it falls back to TargetFramework. We should clean that up in a subsequent PR. Not going to touch this PR as it's currently blocking my work in runtime.

**Pull request merged**